src/index.ts

```diff
@@ -1,60 +1,21 @@
-import * as core from '@actions/core';
-import { Action, Docker, ImageTag, Input, Output, ResultsCheck } from './model';
+import { run as main } from './main';
+import { run as post } from './post';
 
-async function run() {
-  try {
-    Action.checkCompatibility();
+import { basename } from 'path';
 
-    const { workspace, actionFolder } = Action;
-    const {
-      editorVersion,
-      customImage,
-      projectPath,
-      customParameters,
-      testMode,
-      coverageOptions,
-      artifactsPath,
-      useHostNetwork,
-      sshAgent,
-      gitPrivateToken,
-      githubToken,
-      checkName,
-      chownFilesTo,
-    } = Input.getFromUser();
-    const baseImage = new ImageTag({ editorVersion, customImage });
-    const runnerTemporaryPath = process.env.RUNNER_TEMP;
+async function run([_, name]: string[]) {
+  const script = basename(name ?? 'main.js');
```

Comment:
prefer a separate line instead of using null coalesce operator (which can be hard to understand for new developers, especially when having to consider all possible execution paths)

Reply:
Sure.

Reply:
Addressed.